cloud/azure/bin/lib/key_vault.sh

```diff
@@ -124,5 +124,5 @@ function key_vault::has_secret() {
     --query value \
     -o tsv 2>&1 >/dev/null)"
 
-  echo "${SECRET_RESULT}" | grep -q -v "SecretNotFound"
+  echo "${SECRET_RESULT}" | grep "SecretNotFound" | echo $
```

Comment:
What the intent with this line? It's only ever going to print a "$".

Reply:
debugging a bash issue. unfortunately the easiest development flow is to commit a change (thus my long list of commits).